client-config/src/test/java/com/palantir/conjure/java/client/config/ClientConfigurationsTest.java

```diff
@@ -71,7 +71,8 @@ public void testFromParameters_fillsInDefaults() {
         ClientConfiguration actual = ClientConfigurations.of(uris, sslFactory, trustManager);
 
         assertThat(actual.sslSocketFactory()).isInstanceOf(KeepAliveSslSocketFactory.class);
-        assertThat(((KeepAliveSslSocketFactory) actual.sslSocketFactory()).getDelegate()).isEqualTo(sslFactory);
+        assertThat(((KeepAliveSslSocketFactory) actual.sslSocketFactory()).getDelegate())
+                .isEqualTo(sslFactory);
```

Comment:
@iamdanfox @ferozco 0.3.1 made a lot of the fluent assertions break.
I don't think that's necessarily bad though, it makes it a lot more obvious that there is a method call without having to look all the way to the right for it.

Reply:
I think the additional break is fine